components/infobox/wikis/starcraft2/infobox_person_map_maker_custom.lua

```diff
@@ -0,0 +1,101 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:Infobox/Person/MapMaker/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local PersonSc2 = require('Module:Infobox/Person/Custom/Shared')
+local Person = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Class = require('Module:Class')
+
+--race stuff tables
+PersonSc2.raceDisplayLookupTable = {
+	['p'] = '[[Protoss]]',
+	['pt'] = '[[Protoss]],&nbsp;[[Terran]]',
+	['pz'] = '[[Protoss]],&nbsp;[[Zerg]]',
+	['t'] = '[[Terran]]',
+	['tp'] = '[[Terran]],&nbsp;[[Protoss]]',
+	['tz'] = '[[Terran]],&nbsp;[[Zerg]]',
+	['z'] = '[[Zerg]]',
+	['zt'] = '[[Zerg]],&nbsp;[[Terran]]',
+	['zp'] = '[[Zerg]],&nbsp;[[Protoss]]',
+	['r'] = '[[Random]]',
+	['a'] = '[[Protoss]],&nbsp;[[Terran]],&nbsp;[[Zerg]]',
+}
```

Comment:
Both MapMaker and Player seems to override this lookup table. Am I correct in that it's only done because Players should have the categories and MapMaker not?

Reply:
yeah
i will switch it a bit up